src/main/java/reposense/RepoSense.java

```diff
@@ -105,4 +106,14 @@ public static void main(String[] args) {
 
         return configs;
     }
+
+    public static String getVersion() {
+        String version = RepoSense.class.getPackage().getImplementationVersion();
+
+        if (version == null || "unspecified".equals(version)) {
```

Comment:
Abstract out "unspecified" as a constant variable.
Also, since we already did a `version == null` check before, change the order of `equals` -> `version.equals("unspecified")` instead (sounds more logically correct)